graphql/documents/mutations/scene-filter.graphql

```diff
@@ -0,0 +1,77 @@
+mutation SceneFilterCreate(
```

Comment:
I know you've copied this from scene markers, but this would be much cleaner if it accepted a `($input: SceneFilterCreateInput!)`, like the scene mutations, instead of each field individually. The same goes for `SceneFilterUpdate`.